cli/src/rust_template.rs

```diff
@@ -19,21 +18,46 @@ pub enum ProgramTemplate {
     Single,
     /// Program with multiple files for instructions, state...
     Multiple,
+    /// Generate template for Rust unit-test
+    RustTest,
```

Comment:
Why do we need this in `ProgramTemplate`? Tests should be handled in `TestTemplate` if possible.

Reply:
Thank you.
I was a bit of confused with [this previous conversation](https://github.com/coral-xyz/anchor/pull/2805#pullrequestreview-1861039608), but now its clear. I removed `RustTest` variant.